<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EntityWindow</class>
 <widget class="QWidget" name="EntityWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>261</width>
    <height>182</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>221</width>
     <height>74</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QCheckBox" name="WanderBox">
      <property name="text">
       <string>Wander</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="ArriveBox">
      <property name="text">
       <string>Arrive</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="ObstacleBox">
      <property name="text">
       <string>Obstacle Avoidance</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>105</width>
     <height>25</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QDoubleSpinBox" name="massBox"/>
    </item>
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Mass</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>136</width>
     <height>25</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QDoubleSpinBox" name="forceBox"/>
    </item>
    <item>
     <widget class="QLabel" name="forceLabel">
      <property name="text">
       <string>Max Force</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <zorder>layoutWidget</zorder>
  <zorder>WanderBox</zorder>
  <zorder>massBox</zorder>
  <zorder>label</zorder>
  <zorder>forceBox</zorder>
  <zorder>forceLabel</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
